src/System.Threading.Tasks.Parallel/src/System/Threading/Tasks/ParallelRangeManager.cs

```diff
@@ -37,6 +41,7 @@ internal struct IndexRange
     /// <summary>
     /// The RangeWorker struct wraps the state needed by a task that services the parallel loop
     /// </summary>
+    [StructLayout(LayoutKind.Auto)]
```

Comment:
In this case, the struct has the following fields, in this defined order:
- reference
- int
- long
- long
- long
- bool

In a 64-bit process, with sequential layout, the int field will need to be padded so that the longs can be properly aligned, and the bool will end up being expanded to be 8 bytes as well, so the struct will be 48 bytes.  As auto, the runtime can reorder the fields to avoid the extra padding, e.g. putting the int at the end with the bool, and it'll end up being only 40 bytes.